### PR TITLE
Move citations to in-text/in-tutorial rather than topic level metadata

### LINKED ISSUE #10: References
I would also move the references to (a separate file) in each folder in stead of in the general readme. I saw for some material there is already a list of references.

Splitting the markdown files will aid the maintainability IMO but will require merging them to generate the pdf.

### PR DESCRIPTION
**Pull request by erasche**
Fixes #10

### CODE REVIEW

File: topics/chip-seq/metadata.yaml [hunk 1]
```diff
@@ -22,110 +22,110 @@ maintainers:
   - friedue
 
 references:
```
Comment (bebatut) [inline topics/chip-seq/metadata.yaml]: Should we try to remove them here too and move them to the tutorial level?

Comment (erasche) [reply topics/chip-seq/metadata.yaml]: Yes, definitely, just a lot where it wasn't (yet) obvious which tutorial it belonged to or how/where to insert in the text.

File: topics/metabolomics/tutorials/lcms-data-import/tutorial.md [hunk 2]
```diff
@@ -23,10 +23,10 @@ contributors:
 
 **TODO** Explain why metabolomics, what do you want to do
 
-To illustrate this approach, we will use data from [Thévenot et al, 2015](https://pubs.acs.org/doi/10.1021/acs.jproteome.5b00354). The objectives of this paper was to analyze the inﬂuence of age, body mass index, and gender on the urine metabolome. To do that, the authors collected samples from 183 employees from CEA and did
+To illustrate this approach, we will use data from {% cite Thvenot2015 %}. The objectives of this paper was to analyze the inﬂuence of age, body mass index, and gender on the urine metabolome. To do that, the authors collected samples from 183 employees from CEA and did
```
Comment (lecorguille) [inline topics/metabolomics/tutorials/lcms-data-import/tutorial.md]: I think I have understood the idea.
Do you want me to create a `topics/metabolomics/tutorials/lcms-data-import/tutorial.md`?

Comment (erasche) [reply topics/metabolomics/tutorials/lcms-data-import/tutorial.md]: oh, it looks like I forgot to include that one. Thanks, found a couple more I forgot to add :)

**Comment by shiltemann** [comment]
thanks @erasche! ..we can update the rest of the tutorials/topics in a followup PR

**Pull request merged**